scripts/legislation/africa/rwanda_french.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+function remove_all_text_before_first_chapter_header {
+  sed -n '/CHAPITRE PREMIER - DES DISPOSITIONS GENERALES /,$p'
+}
+
+function amend_errors_in_headers {
+  sed -E "s/^(Section [0-9]+ - [A-Za-z,' ]+)[^a-z]+Article ([0-9]+)(er)?/\1\n\n(\2)/" | \
+    sed -E 's/CHAPITRE PREMIER - DES DISPOSITIONS GENERALES/CHAPITRE PREMIER - DES DISPOSITIONS GÉNÉRALES/' | \
+    sed -E 's/Article premier:/\n\n(1)/' | \
+    sed -E 's/^(CHAPITRE )I -l:/\1II -/' | \
+    sed -E 's/CHAPITRE II - DES DEFINITIONS DE QUELQUES MOTS UTILISES DANS LA PRESENTE LOI ORGANIQUE/CHAPITRE II - DES DÉFINITIONS DE QUELQUES MOTS UTILISÉS DANS LA PRÉSENTE LOI ORGANIQUE/' | \
+    sed -E 's/^(TITRE )I -l:/\1II -/' | \
+    sed -E "s/(CHAPITRE PREMIER - DE L'ENVIRONNEMENT NATUREL )/\1\n\n/" | \
+    sed -E 's/^(Section première: Le sol.*) Article 11:/\1\n\n(11)/' | \
+    sed -E 's/^(Section 3 - De la diversité.*) Article 20:/\1\n\n(20)/' | \
+    sed -E "s/^(Section 4 - De l'atmosphère.*) Article 25:/\1\n\n(25)/" | \
+    sed -E "s/^(Section 2 - Obligations.*) Article 60:/\1\n\n(60)/" | \
+    sed -E 's/^(CHAPITRE )I -ll:/\1III -/' | \
+    sed -E 's/^(CHAPITRE IV - .*)LA PRESENTE LOI/\1LA PRÉSENTE LOI/' | \
+    sed -E "s/^(TITRE III - .*)DE L'ETAT, DES COLLECTIVITES/\1DE L'ÉTAT, DES COLLECTIVITÉS LOCALES/" | \
+    sed -E "s/^CHAPITRE PREMIER - DES OBLIGATIONS GENERALES/CHAPITRE PREMIER - DES OBLIGATIONS GÉNÉRALES/" | \
+    sed -E "s/^(CHAPITRE II - DES OBLIGATIONS.*)Section première:/\1\n\nSection première:/" | \
+    sed -E "s/^(CHAPITRE IV - .*)DES ETUDES/\1DES ÉTUDES/" | \
+    sed -E "s/^(TITRE VI - .*)PREVENTIVES/\1PRÉVENTIVES/" | \
+    sed -E "s/^(CHAPITRE PREMIER - DES DISPOSITIONS.*)PREVENTIVES/\1PRÉVENTIVES/" 
+}
+
+function amend_errors_in_articles {   
+    #Article 1
+    amend_error_in_article 1 'bien être' 'bien-être' | \
```

Comment:
I should have been clearer. The first function in the chain of invocations (i.e. the first function in the collection of functions that are piped together) should not be indented further. The further indenting serves to illustrate that the functions below are part of the same statement as the first function in the chain.

Reply:
By "piped together", I mean function invocations that are joined together by the pipe character (`|`), where the output from the function to the left of the pipe becomes the input to the function to the right of the pipe.

Yes, default indent is two spaces and continuation lines should be indented four spaces.